app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
-end
+def full_title(page_title = nil)
```

Comment:
Fix indentation.

You said that this method doesn't works correctly. Something changed?